proposals/stage-1/SUPPORTER_PROGRAM/SUPPORTER_PROGRAM.md

```diff
@@ -0,0 +1,51 @@
+## The People's Community of JavaScriptlandia 
+
+JavaScriptLandia is a place where all JavaScript developers, whether if you're all about TypeScript, a React evangelist, or you stick to Node.js gather to declare their support for the JavaScript ecosystem and OpenJS. 
+
+It's an Individual Membership Program whose goals are to help bring awareness about OpenJS projects to possible contributors. A way for individuals to show support and be a part of the OpenJS foundation, as well as providing an opportunity for community with the other members. 
+
+While one goal of this group is introducing members to projects and helping the projects find the support they need; the membership program will also be an opportunity to welcome an inclusive group of coders from around the world that love the projects that are part of the OpenJS foundation. This also brings in new perspectives and feedback from the vast community of JavaScript developers.
```

Comment:
> find the support they need

This seems a bit vague, it could be financial, technical, other. Might be good to be a bit more specific on this... but this is in no way a blocking comment

Reply:
Yup, I'm going to say "support via new core committers" unless we also think it will net in financial support?

Reply:
sgtm